ui/components/app/currency-input/currency-input.js

```diff
@@ -44,13 +44,16 @@ export default function CurrencyInput({
 
   const [isSwapped, setSwapped] = useState(false);
   const [newHexValue, setNewHexValue] = useState(hexValue);
+  const [newFeatureSecondary, setNewFeatureSecondary] = useState(
+    featureSecondary,
+  );
 
   const shouldUseFiat = () => {
     if (hideSecondary) {
       return false;
     }
 
-    return Boolean(featureSecondary);
+    return Boolean(newFeatureSecondary);
```

Comment:
I think it's the name that's confusing me. In this place `newFeatureSecondary` is used to show either fiat or crypto, but the name doesn't reflect anything like that.

Reply:
@segun [Renamed](https://github.com/MetaMask/metamask-extension/pull/13827/commits/9ea5bb73f19d162b2ffbf0d2764aa14ff2d448e1) as we agreed!